geonode/groups/templates/groups/group_members.html

```diff
@@ -41,6 +41,13 @@ <h4>
                   {% csrf_token %}
                   <button type="submit" class="btn btn-danger btn-xs">{% trans 'Remove' %}</button>
                 </form>
+                {% if member.role == 'member' %}
+                  |
+                  <form style="display: inline;" method="POST" action="{% url "group_member_promote" group.slug member.user.username %}">
+                    {% csrf_token %}
+                    <button type="submit" class="btn btn-success btn-xs">{% trans 'Promote' %}</button>
```

Comment:
If a new translation string is added, should we also update the django.po files? I have done a __very quick__ search for `Promote` as string and have not found it in EN locale.